apis/configuration/slo_controller_config.go

```diff
@@ -275,6 +275,9 @@ type CPUNormalizationStrategy struct {
 	// Enable defines whether the cpu normalization is enabled.
 	// If set to false, the node cpu normalization ratio will be removed.
 	Enable *bool `json:"enable,omitempty"`
+	// Ratio defines the cpu normalization.
+	// If Ratio is set, directly use it.
+	Ratio *float64 `json:"ratio,omitempty"`
```

Comment:
To not break the design of the cpu normalization, perhaps it is a `DefaultRatio` when the RatioModel is missing or the cpu model is unknown.

Reply:
In some cases, there are some dedacated node pools in a cluster. Pods will exhibit their own performance characteristics, and we can set different ratios to meet their respective needs.
The default ratio may not meet the above requirements.

Reply:
It seems the requirements. For your dedicated node pools, you can set the defaultRatio with nodeSelector targeting the node pool's labels. The main difference between the `defaultRatio` and the `ratio` of this commit is that the `RatioModels` take precedence over the `defaultRatio to keep the "cpu normalization" as the goal. In contrast, the `ratio` is just a special field to bypass the models.

Reply:
> It seems the requirements can be satisfied. For your dedicated node pools, you can set the defaultRatio with nodeSelector targeting the node pool's labels. The main difference between the `defaultRatio` and the `ratio` of this commit is that the `RatioModels` take precedence over the `defaultRatio` to keep the "cpu normalization" as the goal. In contrast, the `ratio` is just a special field to bypass the models.

Good idea, I misunderstood your meaning at first. I have changed it to `DefaultRatio`. 
When  the RatioModel is missing or the cpu model is unknown, use the `DefaultRatio`. 
 thanks. cc @saintube